### PR TITLE
Add a test that tests the Bugzilla API with relationships

### PR DESCRIPTION
**Pull request by mprahl**
This also adds the ability to run just a single test

### CODE REVIEW

File: tests/api/test_bugzilla.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+from purview.models.user import User
+from purview.models.bugzilla import BugzillaBug
+from purview.models.distgit import DistGitCommit
+from purview.models.errata import Advisory
+
+
+def test_get_bug(client):
+    """Test getting a resource from Neo4j with its relationships."""
+    tbrady = User.get_or_create({
```
Comment (yashvardhannanavati) [inline tests/api/test_bugzilla.py]: Should we have general functions like 'create_user' and 'create bug' so that we can reduce duplication?

Comment (mprahl) [reply tests/api/test_bugzilla.py]: I was thinking so but it doesn't save much code.

Comment (yashvardhannanavati) [reply tests/api/test_bugzilla.py]: cool

**Comment by yashvardhannanavati** [comment]
:+1:

**Pull request merged**